lib/rubygems/compiler.rb

```diff
@@ -24,18 +24,17 @@ def compile
 
     build_extensions
 
-    artifacts = (@options[:artifacts] || []).map do |rel_path|
-      full_path = File.join(target_dir, rel_path)
-      if File.file?(full_path)
-        full_path
+    artifacts = collect_artifacts
+
+    (@options[:artifacts] || []).map do |include, glob|
+      resolved = Dir.glob("#{target_dir}/#{glob}")
+      if include
```

Comment:
Can I recommend any word other than `include`?

Reply:
Yeah. Missed that one. to_include ? to_be_included ? Or I could make it a `mode` variable that is either `:include` or `:exclude`. What do you think ?